### PR TITLE
Fix uncorrect links and lang code in translation

### PR DESCRIPTION
**Pull request by hzlmn**

### CODE REVIEW

**Comment by arthurvr** [comment]
Those links aren't really incorrect, unquoted attribute values are perfectly valid. Though, it's consistent with the rest of the code style so I'm okay with changing those.

If we decide to change it let's do it in all in once, there are plenty other appearances of that in our codebase.

File: pt-br/index.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html public "i ♥ the web">
 <!--[if lt IE 9]>      <html class="no-js oldie no-fontface" lang="en"> <![endif]-->
```
Comment (arthurvr) [inline pt-br/index.html:2]: this one needs updating as well :)

File: pt-br/index.html [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
-<!--[if gt IE 8]><!--> <html class="no-js no-fontface" lang="en">   <!--<![endif]-->
+<!--[if gt IE 8]><!--> <html class="no-js no-fontface" lang="pt">   <!--<![endif]-->
```
Comment (arthurvr) [inline pt-br/index.html:3]: I wonder if this should be `pt` or `pt-br` as the folder name.

**Comment by drKraken** [comment]
@arthurvr  Yeah, it's not really incorrect, but what i actually mean, that such type of link is used only in this translation, so i think it will be good to keep same style here too.

**Pull request merged**